server/id/id.go

```diff
@@ -45,14 +46,29 @@ type allocatorImpl struct {
 	base uint64
 	end  uint64
 
-	client   *clientv3.Client
-	rootPath string
-	member   string
+	client    *clientv3.Client
+	rootPath  string
+	allocPath string
+	label     string
+	member    string
+	metrics   *metrics
+}
+
+// metrics is a collection of idAllocator's metrics
+type metrics struct {
+	idGauge prometheus.Gauge
 }
 
 // NewAllocator creates a new ID Allocator.
-func NewAllocator(client *clientv3.Client, rootPath string, member string) Allocator {
-	return &allocatorImpl{client: client, rootPath: rootPath, member: member}
+func NewAllocator(client *clientv3.Client, rootPath, allocPath, label, member string) Allocator {
```

Comment:
We need to make allocation step configurable as well. Even though we have 3 bytes reserved for Keyspace ID, 1000 is still too much.